_posts/0000-01-02-Kuldeep-P9.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Sorry this is not the One which i am talking
```suggestion
Your text
Sorry
```

Reply:
But

Reply:
![final](https://user-images.githubusercontent.com/74202807/98577176-90f6a000-22e1-11eb-8d82-647d575f6914.png)

Reply:
**### commit**
_I need this Code For Development _
`Dog`